tests/sentry/workflow_engine/migration_helpers/test_migrate_rule_action.py

```diff
@@ -50,23 +52,54 @@ def assert_action_data_blob(
 
         # If we have a blob type, verify the data matches the blob structure
         if translator.blob_type:
-            for field in translator.blob_type.__dataclass_fields__:
-                mapping = translator.field_mappings.get(field)
-                if mapping:
-                    # For mapped fields, check against the source field with default value
-                    source_value = compare_dict.get(mapping.source_field, mapping.default_value)
-                    assert action.data.get(field) == source_value
-                else:
-                    # For unmapped fields, check directly with empty string default
-                    if action.type == Action.Type.EMAIL and field == "fallthroughType":
-                        # for email actions, the default value for fallthroughType should be "ActiveMembers"
-                        assert action.data.get(field) == compare_dict.get(field, "ActiveMembers")
+            # Special handling for JiraDataBlob which has additional_fields
+            if translator.blob_type == JiraDataBlob:
+                # Get standard fields from JiraDataBlob, excluding additional_fields
+                standard_fields = {
+                    f.name
+                    for f in dataclasses.fields(JiraDataBlob)
+                    if f.name != "additional_fields"
+                }
```

Comment:
If this is something we constantly reference, could we move this logic into the JiraDataBlob class itself?